node/full.go

```diff
@@ -1,26 +0,0 @@
-package node
-
-import (
-	"go.uber.org/fx"
-
-	"github.com/celestiaorg/celestia-node/core"
-	nodecore "github.com/celestiaorg/celestia-node/node/core"
-	"github.com/celestiaorg/celestia-node/node/fxutil"
-	"github.com/celestiaorg/celestia-node/node/rpc"
-	"github.com/celestiaorg/celestia-node/node/services"
-	"github.com/celestiaorg/celestia-node/service/block"
-)
-
-// fullComponents keeps all the components as DI options required to built a Full Node.
-func fullComponents(cfg *Config, repo Repository) fx.Option {
-	return fx.Options(
-		lightComponents(cfg, repo),
-		fxutil.ProvideIf(!cfg.Core.Remote, repo.Core), // provide core repo constructor only in embedded mode.
-		nodecore.Components(cfg.Core),
-		fx.Provide(func(client core.Client) block.Fetcher {
-			return core.NewBlockFetcher(client)
-		}),
-		fx.Provide(rpc.NewServer),
-		fx.Provide(services.Block),
-	)
-}
```

Comment:
Are these not necessary anymore?

Reply:
Block is still needed, but not the RPC rn, it is not used and needs to be reworked anyway.